django_netjsonconfig/static/django-netjsonconfig/js/widget.js

```diff
@@ -67,12 +67,41 @@
         return editor;
     };
 
+    function evaluateVars(data, context){
```

Comment:
try to use the same function definition style, I see here you use `function name() {}` and below you use `var name = function() {}`